### PR TITLE
add note containing useful link

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1402

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/create-groups/manage-creation-of-groups.md [hunk 1]
```diff
@@ -75,6 +75,9 @@ To manage who creates Groups, the following people need Azure AD Premium license
 - The admin who configures these group creation settings
 - The members of the security group who are allowed to create groups
 
+> [!NOTE]
+> Check [this article](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/license-users-groups) for more details about how to assign Azure licenses.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/manage-creation-of-groups.md]:
```suggestion
> See [Assign or remove licenses in the Azure Active Directory portal](https://docs.microsoft.com/azure/active-directory/fundamentals/license-users-groups) for more details about how to assign Azure licenses.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@MikePlumleyMSFT - PR has been copy edited and is ready for final review. This is based on issue #1402.

Thank you.

cc: @kenwith

**Comment by mypil** [comment]
@MikePlumleyMSFT - PR has been copy edited and is ready for final review. This is based on issue #1402.

Thank you.

cc: @kenwith

**Pull request merged**